langchain/agents/initialize.py

```diff
@@ -51,7 +51,7 @@ def initialize_agent(
             )
         agent_cls = AGENT_TO_CLASS[agent]
         agent_obj = agent_cls.from_llm_and_tools(
-            llm, tools, callback_manager=callback_manager
+            llm, tools, callback_manager=callback_manager, **kwargs
```

Comment:
these are the same kwargs also passed to the AgentExecutor though right? thats kind of an issue...

could we add another parameter that is `agent_kwargs = Optional[dict] = None` that we pass through here

Reply:
done